Failure # 1 (occurred at 2025-01-27_15-02-26)
[36mray::ImplicitFunc.train()[39m (pid=113989, ip=10.12.92.124, actor_id=79dba6a28f67071834f385fd01000000, repr=ray_tune_trainable)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/scs_deal_learning/users/mchowdhu/OptimisingWeightUpdateHyperparameters/envOWHP/lib/python3.11/site-packages/ray/tune/trainable/trainable.py", line 375, in train
    raise skipped from exception_cause(skipped)
  File "/home/scs_deal_learning/users/mchowdhu/OptimisingWeightUpdateHyperparameters/envOWHP/lib/python3.11/site-packages/ray/tune/trainable/function_trainable.py", line 349, in entrypoint
    return self._trainable_func(
           ^^^^^^^^^^^^^^^^^^^^^
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/scs_deal_learning/users/mchowdhu/OptimisingWeightUpdateHyperparameters/envOWHP/lib/python3.11/site-packages/ray/tune/trainable/function_trainable.py", line 666, in _trainable_func
    output = fn()
             ^^^^
  File "/home/scs_deal_learning/users/mchowdhu/OptimisingWeightUpdateHyperparameters/parallel_exec.py", line 275, in ray_tune_trainable
    train.main(config_dict=master_config, config_override=repetition_override)
  File "/home/scs_deal_learning/users/mchowdhu/OptimisingWeightUpdateHyperparameters/train.py", line 868, in main
    learner = Learner(tensorboard_tracker=tracker,
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/scs_deal_learning/users/mchowdhu/OptimisingWeightUpdateHyperparameters/train.py", line 96, in __init__
    raise KeyError("'model' configuration is missing. Ensure the 'model' key is present in your configuration.")
KeyError: "'model' configuration is missing. Ensure the 'model' key is present in your configuration."
